app/src/main/java/grinnell/appdev/edu/lyles/menuItems/BeerItem.java

```diff
@@ -1,16 +1,18 @@
 package grinnell.appdev.edu.lyles.menuItems;
 
 /**
- * Model beer item. Contains
+ * Model beer item. Contains information on beer name and beer description
  */
 public class BeerItem {
 
     private String mName;
     private String mDescription;
+    private double mPrice;
 
     public BeerItem() {
         mName = null;
         mDescription = null;
+        mPrice = 0.00;
```

Comment:
What is 0.00 ? Is it some magic number? Please don't use magic numbers. Btw, you should just leave this as a default constructor. All the fields will be at their default values.